src/util/symbol_table_base.cpp

```diff
@@ -60,6 +60,16 @@ void symbol_table_baset::show(std::ostream &out) const
     out << symbols.at(name);
 }
 
+symbol_table_baset::const_iteratort symbol_table_baset::end() const
```

Comment:
I’m not sure this change is even necessary; Develop `symbol_table_baset` already seems to have `begin()/end()`.

Reply:
It has a non-const version - this adds a const version